pkg/apis/serving/v1beta1/explainer_aix360.go

```diff
@@ -44,10 +45,20 @@ type AIXExplainerSpec struct {
 var _ ComponentImplementation = &AIXExplainerSpec{}
 
 func (s *AIXExplainerSpec) GetStorageUri() *string {
-	if s.StorageURI == "" {
-		return nil
+	if s.StorageURI != "" {
+		return &s.StorageURI
+	} else if s.Storage != nil && s.Storage.StorageKey != nil && s.Storage.Path != nil {
+		storageSpecUri := "secret://" + *s.Storage.StorageKey + "/" + strings.TrimPrefix(*s.Storage.Path, "/")
+		return &storageSpecUri
+	}
+	return nil
+}
```

Comment:
@Tomcli I don't think @pvaneck's first iteration of the `ServingRuntime` stuff is touching this as you thought it might, so as discussed I think we should just move these func implementations to `PredictorExtensionSpec` and `ExplainerExtensionSpec`.

Reply:
done